Code/RDBoost/Wrap.h

```diff
@@ -370,4 +370,53 @@ struct rdkit_pickle_suite : python::pickle_suite {
   static bool getstate_manages_dict() { return true; }
 };
 
+template<typename T>
+void addToList(python::list &l, const T &v) {
+  l.append(v);
+}
+
+template<typename T>
+void addToList(python::list &l, const std::vector<T> &v) {
+  l.append(python::list(v));
```

Comment:
This should probably be 
```
addToList<T>(l, v)
```

And then we could get rid of the std::vector<std::vector<T>> funds